lib/src/navbar/navbar-menu/navbar-menu.component.scss

```diff
@@ -1,11 +1,13 @@
 @import '../../colors.scss';
 
+$menu-z-index: 1025 !default;
```

Comment:
Z-index variables are wise. :+1: Down the road might be good to have a z-index file that helps us track all our z-index's in one place. This can help squash bugs before they happen.

Reply:
Agreed - I saw @benjanderson doing this on `hc-navbar` and it looked like good practice.  There are now three components in the header that have to maintain their correct z-index, the navbar, subnavbar, and responsive menu.  So that's a great idea down the road to collect those all into one place.